magenta/interfaces/midi/midi.py

```diff
@@ -247,25 +264,34 @@ class MonoMidiPlayer(threading.Thread):
     _outport: The Mido port for sending messages.
     _sequence: The monohponic, chronologically sorted NoteSequence to play.
     _stop_playback: A boolean specifying whether the playback should stop.
+    _bpm: The tempo
   Args:
     outport: The Mido port for sending messages.
     sequence: The monohponic, chronologically sorted NoteSequence to play.
   """
   daemon = True
 
+  if not 0 <= FLAGS.metronome_playback_mix <= 100:
+      raise ValueError(
+          'The metronome_playback_mix must be between 0 and 100')
+
   def __init__(self, outport, sequence):
     self._outport = outport
     self._sequence = sequence
     self._stop_playback = False
+    self._bpm = sequence.tempos[0].bpm
```

Comment:
nope, you're right. Also, if there are multiple tempos is it only because of a tempo change during the capture of the NoteSequence? And would sequence.tempos[-1].bpm be the latest tempo or the starting tempo (or something else)?